lib/commands/deploynodeapp.js

```diff
@@ -72,6 +78,35 @@ var descriptor = defaults.defaultDescriptor({
     name: 'Preserve policies from previous revision',
     shortOption: 'P',
     toggle: true
+  },
+  runtime: {
+    name: 'Runtime to deploy the Node.js app to',
+    shortOption: 'B',
```

Comment:
Same comment as below, there are few option shorthands left to pick from that made sense to me.